ipatests/test_integration/test_trust.py

```diff
@@ -992,3 +992,90 @@ def test_server_option_with_unreachable_ad(self):
             self.master.run_command(['rm', '-f', ad_zone_file])
             tasks.configure_dns_for_trust(self.master, self.ad)
             self.remove_trust(self.ad)
+
+
+class TestNonPosixADTrustWithAutoPrivateGroup(BaseTestTrust):
+    """
+    Tests for non-posix AD trust with various options for
+    auto-private-groups
+    """
+
+    def test_establish_nonposix_trust(self):
+        tasks.configure_dns_for_trust(self.master, self.ad)
+        tasks.establish_trust_with_ad(
+            self.master, self.ad_domain,
+            extra_args=['--range-type', 'ipa-ad-trust'])
+
+    def test_idrange_mod_auto_private_group_false(self):
+        range_name = self.ad_domain.upper() + '_id_range'
+        self.master.run_command(['ipa', 'idrange-show', range_name])
+        cmd = self.master.run_command(
+            ['ipa','idrange-mod', range_name, '--auto-private-groups=false'])
+        assert 'Auto private groups: false' in cmd.stdout_text
+        self.master.run_command(['systemctl', 'restart', 'sssd.service'])
+
+    def test_idrange_mod_auto_private_group_true(self):
+        range_name = self.ad_domain.upper() + '_id_range'
+        self.master.run_command(['ipa', 'idrange-show', range_name])
+        cmd = self.master.run_command(
+            ['ipa','idrange-mod', range_name, '--auto-private-groups=true'])
+        assert 'Auto private groups: true' in cmd.stdout_text
+        self.master.run_command(['systemctl', 'restart', 'sssd.service'])
+
+    def test_idrange_mod_auto_private_group_hybrid(self):
+        range_name = self.ad_domain.upper() + '_id_range'
+        self.master.run_command(['ipa', 'idrange-show', range_name])
+        cmd = self.master.run_command(
+            ['ipa','idrange-mod', range_name, '--auto-private-groups=hybrid'])
+        assert 'Auto private groups: hybrid' in cmd.stdout_text
+        self.master.run_command(['systemctl', 'restart', 'sssd.service'])
+
+
+class TestPosixADTrustWithAutoPrivateGroup(BaseTestTrust):
+    """
+    Tests for Posix AD trust with various options for
+    auto-private-groups
+    """
+
+    def test_establish_nonposix_trust(self):
+        tasks.configure_dns_for_trust(self.master, self.ad)
+        tasks.establish_trust_with_ad(
+            self.master, self.ad_domain,
+            extra_args=['--range-type', 'ipa-ad-trust-posix'])
+
+    def test_idrange_mod_auto_private_group_false(self):
+        range_name = self.ad_domain.upper() + '_id_range'
+        self.master.run_command(['ipa', 'idrange-show', range_name])
+        cmd = self.master.run_command(
+            ['ipa','idrange-mod', range_name, '--auto-private-groups=false'])
```

Comment:
no space between (`ipa` and `idrange-mode`) >  `['ipa', 'idrange-mod', range_name, '--auto-private-groups=false']`
Wondering why pep8 check didn;t catch!